transmogrifier/sources/datacite.py

```diff
@@ -117,15 +118,39 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
 
         for date in xml.metadata.find_all("date"):
             d = timdex.Date()
-            if value := date.string:
-                if "/" in value:
-                    split = value.index("/")
-                    d.range = timdex.Date_Range(
-                        gte=value[:split],
-                        lte=value[split + 1 :],
-                    )
+            if date_value := date.string:
```

Comment:
Rather than continuing to use this check, let's update the `find_all` function above to include `string=True`.

Reply:
Ah got it, yeah that makes sense.